frontend/src/components/where-buy/where-buy-component/styles.module.scss

```diff
@@ -0,0 +1,161 @@
+@import "@assets/css/variables";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 15px 25px;
+  background-color: var(--white);
+  border: solid 1px var(--white);
+  border-radius: 5px;
+
+  .logoDiv {
+    display: flex;
+    align-items: center;
+    width: 93px;
+    margin-right: 15px;
+    padding: 0 3px 0 0;
+
+    .logo {
+      width: 90px;
+      margin: 0;
+    }
+  }
+
+  .description {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    width: auto;
+
+    .carName {
+      margin-bottom: 5px;
+      font-weight: 600;
+      font-size: 14px;
+      text-transform: uppercase;
+    }
```

Comment:
The same. Font-size in rem better.
And all next places in PR where font-size in px.